client/packages/host/src/components/Navigation/ProgramsNav.tsx

```diff
@@ -19,12 +19,13 @@ export const ProgramsNav = ({ store }: { store?: UserStoreNodeFragment }) => {
   );
   const t = useTranslation('app');
   const isCentralServer = useIsCentralServerApi();
-  const visible = isCentralServer && store?.preferences.vaccineModule;
+  const immunisationsVisible = isCentralServer && store?.preferences.vaccineModule;
+  const rnrVisible = store?.preferences.omProgramModule;
```

Comment:
Does everyone with the omProgramModule get R&R Forms?
Will we add an extra pref or something?

Reply:
Chatted with Mark earlier, we thought just program module for now? Could always hide further if needed (though we seem to have the pattern of "show people what they could be using")